### PR TITLE
Update libs related to Connect

### PR DESCRIPTION
**Pull request by karliatto**
## Related Issue

Resolve

### CODE REVIEW

**Comment by mroz22** [comment]
please lets do it one lib - one commit. including changes in yarn.lock 🙏 makes it easier to review.

**Comment by karliatto** [comment]
> please lets do it one lib - one commit. including changes in yarn.lock 🙏 makes it easier to review.

Yes, I am doing like that and one by one letting all the tests run in CI after updating dependency to see if any breaks something.

**Comment by karliatto** [comment]
> * @SocketSecurity ignore npm/bundle-name@4.1.0

It is dependency of a dependency so even it is a trivial package we would need to update it upstream in order to update it. So @SocketSecurity ignore npm/bundle-name@4.1.0

**Comment by karliatto** [comment]
@SocketSecurity ignore npm/bundle-name@4.1.0

File: packages/address-validator/package.json [hunk 1]
```diff
@@ -20,6 +20,6 @@
     },
     "devDependencies": {
         "jest-environment-jsdom": "29.7.0",
-        "rimraf": "^5.0.5"
+        "rimraf": "^6.0.1"
```
Comment (mroz22) [inline packages/address-validator/package.json:23]: todo later: maybe packages that are already defined in root package json shoudln't be present in particular packages package.jsons? cc @Nodonisko

**Review by mroz22 — APPROVED** [review]

File: packages/address-validator/package.json (hunk 1 above)
Comment (komret) [reply packages/address-validator/package.json:23]: They shouldn't.

Comment (mroz22) [reply packages/address-validator/package.json:23]: ok, @karliatto please just rebase and let's merge it now. Removing root defined packages should be done as a followup.

**Pull request merged**